app/services/database.py

```diff
@@ -107,8 +108,10 @@ def create_audio_file_entry(self, model_info):
         try:
             # db_server = self.global_utility.get_database_server_name()
             # db_name = self.global_utility.get_database_name()
-            db_server = 'FLM-VM-COGAIDEV'
-            db_name = 'AudioTrans'
+            # db_server = 'FLM-VM-COGAIDEV'
+            db_server = os.environ.get("SERVER_NAME")
+            db_name = os.environ.get("DB_NAME")
+            # db_name = 'AudioTrans'
             dns = f'mssql+pyodbc://{db_server}/{db_name}?driver=ODBC+Driver+17+for+SQL+Server'
```

Comment:
Database string can't be hardcoded partially.

Reply:
ODBC driver is also configurable reflected on this changeset e1ae681